### PR TITLE
[NCL-1754] fixed Last build status in the error log

### PR DESCRIPTION
**Pull request by michalszynkiewicz**

### CODE REVIEW

File: build-coordinator/src/main/java/org/jboss/pnc/coordinator/builder/datastore/DatastoreAdapter.java [hunk 1]
```diff
@@ -72,7 +73,9 @@ public BuildConfigSetRecord saveBuildConfigSetRecord(BuildConfigSetRecord buildC
      * @return The latest revision of the given build configuration
      */
     public BuildConfigurationAudited getLatestBuildConfigurationAudited(Integer buildConfigurationId) {
-        return datastore.getLatestBuildConfigurationAudited(buildConfigurationId);
+        BuildConfigurationAudited buildConfigAudited = datastore.getLatestBuildConfigurationAudited(buildConfigurationId);
+        buildConfigAudited.getProject().getBuildConfigurations().forEach(BuildConfiguration::getId);
```
Comment (matejonnet) [inline build-coordinator/src/main/java/org/jboss/pnc/coordinator/builder/datastore/DatastoreAdapter.java]: Why is this loop required here ?

Comment (michalszynkiewicz) [inline build-coordinator/src/main/java/org/jboss/pnc/coordinator/builder/datastore/DatastoreAdapter.java]: I've just tested it. 

`getBuildConfigurations()` is not enough, `LazyInitializationException` is still thrown on accessing the `buildConfigurations` of `Project`.

Long story: AFAIR, we put the stuff that is fetched here in `submittedBuilds`. Then, later, handling a different request, we take a build from it and try to get build configurations from the build.

Comment (matejonnet) [inline build-coordinator/src/main/java/org/jboss/pnc/coordinator/builder/datastore/DatastoreAdapter.java]: @pgier @vibe13 have dealt with some LazyIE we already had.

Comment (michalszynkiewicz) [inline build-coordinator/src/main/java/org/jboss/pnc/coordinator/builder/datastore/DatastoreAdapter.java]: @matejonnet I don't understand. Are their changes in some other PRs and I can rebase on them?

Comment (vibe13) [inline build-coordinator/src/main/java/org/jboss/pnc/coordinator/builder/datastore/DatastoreAdapter.java]: @michalszynkiewicz @matejonnet  If we are going to keep the LAZY fetching, I saw an earlier implementation. It basically does not get rid of the loop, but it's inside a dedicated class:  pnc/rest/src/main/java/org/jboss/pnc/rest/utils/HibernateLazyInitializer.java used inside  pnc/rest/src/main/java/org/jboss/pnc/rest/trigger/BuildTriggerer.java

**Pull request merged**